bin/kano-uixinit

```diff
@@ -193,6 +195,10 @@ if [ -e $startvnc ]; then
     $startvnc
 fi
 
+# TODO: This should be in a drop in directory,
+# installed from the updater instead
+sudo kano-updater ui boot-window
+
```

Comment:
It was because if you scheduled an update, the updater window would pop up asking you to update *before* vnc started. Which meant that you had to close the dialog before your vnc connected.